### ISSUE TITLE
Herecomment after escaped regex does not convert to /* comment */

### ISSUE THREAD

**Issue by netei** [eventualbuddha/decaffeinate#95]
Input : 

```
"foo".replace(/[\/]/g, "/")

###This is a herestring
###
```

Output: 

```
"foo".replace(/[\/]/g, "/");

###This is a herestring
###
```

Expected:

```
"foo".replace(/[\/]/g, "/");

/*This is a herestring
*/
```

**Comment by eventualbuddha** [comment]
Thanks for reporting all the bugs! This is probably similar to some of your others in that it involves my pseudo-lexers within the project, `trimmedNodeRange` being one and [`rangesOfComments`](https://github.com/eventualbuddha/decaffeinate/blob/master/src/utils/rangesOfComments.js) being another. This is probably a good place for someone looking to help out to start.

cc @jeffchan @nickdima

**Comment by eventualbuddha** [comment]
Fixed in v1.41.3.
